editor/src/components/canvas/canvas-strategies/absolute-move-strategy.ts

```diff
@@ -25,6 +25,7 @@ import { wildcardPatch } from '../commands/wildcard-patch-command'
 import { runLegacySnapping } from '../controls/guideline-helpers'
 import { ConstrainedDragAxis, Guideline, GuidelineWithSnappingVector } from '../guideline'
 import { CanvasStrategy } from './canvas-strategy-types'
+import { getMoveCommandsForSelectedElement } from './shared-move-strategy-helpers'
```

Comment:
Either the file name or the imported function should be updated to reflect that this is for absolute positioning

Reply:
I renamed the file